fastlane_bot/helpers/txhelpers.py

```diff
@@ -214,17 +214,17 @@ def _get_prices_info(
             expected_profit_usd: Decimal,
             expected_profit_eth: Decimal,
             signed_arb_tx
```

Comment:
Could you change the param signed_arb_tx to raw_transaction?

I think it will be more clear as the input to this function is: signed_arb_tx.rawTransaction